yb-voyager/cmd/live_migration.go

```diff
@@ -117,6 +120,11 @@ func streamChangesFromSegment(segment *EventQueueSegment, evChans []chan *tgtdb.
 
 		if event == nil && segment.IsProcessed() {
 			break
+		} else if event.IsCutover() && tconf.TargetDBType == YUGABYTEDB ||
```

Comment:
I think the logic should be sometihng like this - 
```
If cutover:
    if importDestinationType = TARGET (use importDestinationTYpe instead of inferring from targetDBType)
        break out of everything. print report and exit
    if importDestinationType = FF 
        ignore, and continue.
```

Reply:
yeah since now we have this `importDestinationType` this makes sense. Will do the change.

Reply:
changed the logic to use that:
```
if event.IsCutover() && importDestinationType == TARGET_DB ||
			event.IsFallForward() && importDestinationType == FF_DB
```

printing of report is done at other step, here just need to return